src/main/java/duke/Parser.java

```diff
@@ -36,6 +27,8 @@ public static Command parse(String input) throws DukeException {
             return new MarkCommand(Integer.parseInt(remaining));
         } else if (input.matches("unmark [1-9]+\\d*")) {
             return new UnmarkCommand(Integer.parseInt(remaining));
+        } else if (firstWord.equals("find")) {
```

Comment:
Might want to consider using enum to replace the String values to make it neater and easier to manage!